app/models/like.rb

```diff
@@ -1,4 +1,8 @@
 class Like < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  def update_likes(count)
+    post.update(like_counter: count)
+  end
 end
```

Comment:
- [ ] Kindly consider making use of one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_likes` method every time a new like has been saved to the database, then make the `update_likes` method private

e.g

```ruby
 after_save :update_likes

  private

  def update_likes
    author.increment!(like_counter)
  end
```